package.json

```diff
@@ -46,7 +46,7 @@
     "xrpl-client": "1.8.0",
     "xrpl-orderbook-reader": "0.3.3",
     "xrpl-tagged-address-codec": "0.2.1",
-    "xumm-string-decode": "0.6.0"
+    "xumm-string-decode": "0.6.1"
```

Comment:
@N3TC4T This is amazing, and so useful to our team. Dinner or drinks on us when we meet in person one day!
@nkramer44 @theotherian